src/nvim/memory.c

```diff
@@ -222,6 +222,21 @@ void *xmemdupz(const void *data, size_t len)
   return memcpy(xmallocz(len), data, len);
 }
 
+/// Duplicates `len` bytes of `src` to `dst` and zero terminates it.
+/// and returns a pointer to the allocated memory. If the allocation fails,
+/// the program dies.
```

Comment:
Is this actually true? I see that this docstring was copied from `xmemdupz`, which calls `xmallocz` which will terminate the program. But this new function does not call anything that would kill if allocation fails.

Reply:
Oops, forgot to delete these two lines.